docs/how_to.md

```diff
@@ -289,6 +290,32 @@ this extension. If the extension is not present or has value 'false' then the re
 *Note: This extension is only interpreted and handled in resource root POST operations (e,g: /v1/resource) in the
 above example*
 
+###### <a name="xTerraformResourceTimeout">x-terraform-resource-timeout</a>
+
+This extension allows service providers to override the default timeout value for CRUD operations with a different value.
+
+The value must comply with the duration type format. A duration string is a sequence of decimal positive numbers (negative numbers are not allowed), 
+each with optional fraction and a unit suffix, such as "300s", "20.5m", "1.5h" or "2h45m".
```

Comment:
I question the value of allowing fractions, not sure it adds anything - but it looks like it might be more work to remove than leave in so ok.

Reply:
yeah we are using the ParseDuration so we get that for free pretty much.